pyproject.toml

```diff
@@ -39,106 +39,126 @@ server = [
 ]
 test = [
   "black==23.12.1",
+  "codespell==2.2.6",
   "flake8==7.0.0",
   "flake8-docstrings==1.7.0",
   "isort==5.13.2",
   "mypy==1.8.0",
+  "pre-commit==3.6.0",
+  "pre-commit-hooks==4.5.0",
   "pylint==3.0.3",
   "pytest==7.4.4",
   "pytest-aiohttp==1.0.5",
-  "pytest-cov==4.1.0",
+  "ruff==0.2.1",
+  "safety==3.0.1"
 ]
 
 [project.scripts]
 matter-server = "matter_server.server.__main__:main"
 
-[tool.codespell]
-ignore-words-list = "pase,"
+[tool.setuptools]
```

Comment:
I think we should sort this whole file :abcd: to make it easier to find things. My vscode sorts the the subitems automatically when I save a pyproject.toml file. It doesn't sort the top level items though. But I'd like for us to sort them manually if we go ahead here.

Reply:
do you know the name of the plugin you use for that ?

Reply:
This one:
https://marketplace.visualstudio.com/items?itemName=be5invis.toml